.github/workflows/database-backup.yml

```diff
@@ -115,3 +115,68 @@ jobs:
           name: backup_sanitised
           path: backup_sanitised.sql
           retention-days: 3
+
+    job2:
+      name: Restore backup to Preprod
+      runs-on: ubuntu-latest
+      needs: [job1]
+      services:
```

Comment:
We don't need the postgres service as it's only used for the sanitisation and it's already done